pegasus/sites.v3/code.org/public/yourschool.haml

```diff
@@ -25,7 +25,9 @@ social:
 - census_announcement['school_zip'] = request.params["zip"] if request.params["zip"]
 - census_announcement['school_year'] = current_census_year
 - census_announcement['locale'] = request.locale
-- census_announcement['teacher_applications_active'] = DCDO.get("teacher_applications_active", CDO.default_teacher_applications_active).to_s
+- accepting_teacher_apps_status  = DCDO.get("accepting_teacher_apps_status", CDO.default_accepting_teacher_apps_status)
+- census_announcement['show_professional_learning_banner'] = (request.locale == 'en-US' && (accepting_teacher_apps_status == 'open' || accepting_teacher_apps_status == 'closing-soon')).to_s
```

Comment:
I'm on the fence about this, to show PL banners, I check if the status is either `open` or `closing-soon`. It seemed safer to explicitly check those two values rather than check if not `closed` in case someone sets the DCDO value to undefined, null or false instead of "closed". But maybe this isn't something I need to worry about?

Reply:
I like it -- maybe just something like `%w(open closing-soon).include? accepting_teacher_apps_status` might improve readability?